packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift

```diff
@@ -225,6 +225,10 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         callback(.success("matt"))
     }
 
+    func gutenbergDidRequestXpost(callback: @escaping (Result<String, NSError>) -> Void) {
+        callback(.success("ma.tt"))
```

Comment:
same as https://github.com/WordPress/gutenberg/pull/26211/files#r540769080, ios demo app didnt seem to have "mocked" cross posts working, perhaps this is just a placeholder.

Reply:
Thanks! Addressed on iOS in bb6ecb5. 

I'm not very familiar with the setup of the iOS demo app, so @guarani , if you would take a look at that commit and make sure there's nothing else I needed to do, I would appreciate it.

Reply:
I think that looks good, thanks for updating that @mchowning!